modular_skyrat/modules/modular_items/lewd_items/code/lewd_quirks.dm

```diff
@@ -50,10 +50,10 @@
 
 /datum/brain_trauma/special/nymphomania
 	name = "Nymphomania"
-	desc = "The patient constantly feels aroused and supposed to satisfy their sexual desires."
-	scan_desc = "constant sexual arousal"
-	gain_text = "<font color=purple>You feel yourself much more hornier than before...</font>"
-	lose_text = "<span class='notice'>A pleasant coolness spreads through the body. You are in control of your sexual desires again.</font>"
+	desc = "The patient constantly feels sexual arousal and heightened libido."
+	scan_desc = "severly heightened libido"
+	gain_text = "<font color=purple>You feel yourself much hornier than before...</font>"
+	lose_text = span_notice("A pleasant coolness spreads through the body. You are in control of your sexual desires again.")
```

Comment:
If you're going to make modifications to nymphomania, please make it so it's no longer a brain trauma. It being a brain trauma is not a good practice in the slightest and leads to inconsistencies and problems in-game associated with those traumas.

Reply:
I don't think there's a way for it to NOT be a brain trauma